test/test_queue_routes.py

```diff
@@ -96,43 +106,88 @@ def test_add_and_edit_task(client, add_sample, add_task):
                      data=json.dumps(task_to_update),
                      content_type='application/json'
                      )
-    assert (rv.status_code == 200 and json.loads(rv.data).get('parameters')[parameter_to_update] == parameter_update_value)
+    assert (rv.status_code == 200 and json.loads(rv.data).get(
+        'parameters')[parameter_to_update] == parameter_update_value)
+
 
 def test_queue_start(client, add_sample, add_task):
-    """Test if we can start the queue. The queue requires a sample and a task to start which are added by fixtures."""
+    """
+    Test if we can start the queue. 
+    The queue requires a sample and a task to start which are added by fixtures. 
+    It also requires a 3d point to be saved before it move from paused state to running. 
+    Unpause is called to mimick that.
+    """
+    rv = client.get('/mxcube/api/v0.1/queue')
+    assert (rv.status_code == 200 and len(
+        json.loads(rv.data).get('1:05')['tasks']) == 1)
+
     rv = client.put('/mxcube/api/v0.1/queue/start',
-                    data=json.dumps({'sid':'1:05'}),
+                    data=json.dumps({'sid': '1:05'}),
                     content_type='application/json')
     assert rv.status_code == 200
 
+    rv = client.put('/mxcube/api/v0.1/queue/unpause')
+    assert rv.status_code == 200
+
+    time.sleep(2)
+    rv = client.get('/mxcube/api/v0.1/queue_state')
+    assert (rv.status_code == 200 and json.loads(
+        rv.data).get('queueStatus') == 'QueueRunning')
+
```

Comment:
What do you think about making sure that the queue also executes to the end, I.E waiting for the queue and then making sure the queue is stopped or even that all collections passed, or is it better to do that in a separate test ?

Reply:
can we leave that for a later PR? I would say that such case is more about internal hwobj behavior, while these tests are more API related.

Reply:
I think it would be a pity to wait and not add it while we are working on it, but I understand that you might be to busy to do it right now. So I'll merge this if you don't have anything else to add ?

Reply:
I would say yes, please merge this as it is for now.